image/preseed/preseed.go

```diff
@@ -54,7 +53,10 @@ type CoreOptions struct {
 	// prepare image directory
 	PrepareImageDir string
 	// key to sign preseeded data with
-	PreseedSignKey string
+	PreseedSignKey *asserts.PrivateKey
+	// assertions for the signing key and its associated account
+	PreseedAccountAssert    *asserts.Account
+	PreseedAccountKeyAssert *asserts.AccountKey
```

Comment:
do we actually fully need both of them? in the end the code will fetch them, we could just reorg/explode a bit what happens in writePreseedAssertion